db/fns.md

```diff
@@ -93,13 +93,15 @@
    * [`get_dependent_tasks`](#get_dependent_tasks)
    * [`get_queue_artifact`](#get_queue_artifact)
    * [`get_queue_artifacts_paginated`](#get_queue_artifacts_paginated)
-   * [`get_task_group`](#get_task_group)
+   * [`get_task_group_project_ids`](#get_task_group_project_ids)
+   * [`get_task_group2`](#get_task_group2)
```

Comment:
Is there a way to avoid having the 2 in the name?

Reply:
this is mostly same function but with changed return signature. I've seen few other functions were using same approach. 
Technically we can change to something like `get_task_group_full` but I think it's fine.